pkg/util/uint256.go

```diff
@@ -93,3 +95,13 @@ func (u Uint256) MarshalJSON() ([]byte, error) {
 // -1 implies u < other.
 //  0 implies  u = other.
 func (u Uint256) CompareTo(other Uint256) int { return bytes.Compare(u[:], other[:]) }
+
+// EncodeBinary implements io.Serializable interface.
+func (u Uint256) EncodeBinary(w *io.BinWriter) {
+	w.WriteBE(u)
```

Comment:
Why `BE`?

Reply:
Actually, it was BE all the way along because of how `binary.Put` for `[]byte` is implemented https://golang.org/src/encoding/binary/binary.go?s=7469:7533#L244

This way it is consistent with other standard `uint*` types.